pytest_playwright/pytest_playwright.py

```diff
@@ -18,6 +18,10 @@
 from typing import Any, Callable, Dict, Generator, List, Optional
 
 import pytest
+from pytest import Item
+from _pytest.config import Config
```

Comment:
I don't see where pytest recommends using their private types. Is there a link?

Reply:
It is used by the official pytest plugins as you can see here https://github.com/pytest-dev/pytest-print/blob/main/src/pytest_print/__init__.py

Reply:
Yeah, I can see how pytest can use its own private apis. But we are not pytest, so we can't do that.